tarball-ingester-iam.tf

```diff
@@ -166,6 +166,18 @@ data "aws_iam_policy_document" "minio_credentials_secretsmanager" {
   }
 }
 
+data "aws_iam_policy_document" "tarball_ingester_describe_autoscaling" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "autoscaling:DescribeAutoScalingInstances",
+    ]
+
+    resources = ["*"]
```

Comment:
does it need to be "*" not sure if you can make it fine grained?

Reply:
tried it finegrained, it didn't like it, hence using *. as discussed with @dan-hill2802 it's a minimal risk.